docs/admin/memory.rst

```diff
@@ -24,29 +24,28 @@ turned on by default.
 
 .. _memory-scopes:
 
-Translation memory scopes
+Translation-memory scopes
 -------------------------
 
 .. versionadded:: 3.2
 
    In earlier versions translation memory could be only loaded from a file
    corresponding to the current imported translation memory scope.
 
-The translation memory scopes are there to allow both privacy and sharing of
-translations, to suit the desired behavior.
+The translation memory scopes allow both privacy and sharing translations.
```

Comment:
```suggestion
The translation memory scopes sharing translations with only relevant components, and projects, or for the entire platform.
```
Does it allow sharing between teams, users, or something else too?

Reply:
This part of the article describes only that there are several scopes. Their feature are described further down, so it is not needed to list it here.

Reply:
It is quite difficult to understand what "translation memory scopes" are.
Something that allows for privacy, and sharing translations somehow doesn't clear that up.